From f0be99ccfdb041f2a4b9f5a03fdd2b68400635bd Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:07:24 +0800
Subject: [PATCH] Test case6 for Add PCD_VAR_CHECK_GENERATION under Define
 section

Description: Add PCD_VAR_CHECK_GENERATION under Define section ("PCD_VAR_CHECK_GENERATION = TRUE") in TestPkg/TestPkg32X64.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 532d084223..3d298053d4 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -23,6 +23,7 @@
   BUILD_TARGETS                  = NOOPT|DEBUG|RELEASE
   SKUID_IDENTIFIER               = DEFAULT
   FLASH_DEFINITION               = TestPkg/TestIa32X64.fdf
+  PCD_VAR_CHECK_GENERATION       = TRUE
 
   #
   # Defines for default states.  These can be changed on the command line.
-- 
2.27.0.windows.1

